templates/flavors/clusterclass/kubeadm-controlplane-template.yaml

```diff
@@ -50,14 +50,12 @@ spec:
         initConfiguration:
           nodeRegistration:
             kubeletExtraArgs:
-              azure-container-registry-config: /etc/kubernetes/azure.json
-              cloud-provider: external
+                cloud-provider: external
             name: '{{ ds.meta_data["local_hostname"] }}'
         joinConfiguration:
           nodeRegistration:
             kubeletExtraArgs:
-              azure-container-registry-config: /etc/kubernetes/azure.json
-              cloud-provider: external
+                cloud-provider: external
```

Comment:
/hold cancel

Reply:
@CecileRobertMichon It looks like this is still indented more than it needs to be. Or is that not actually causing any issues?

Reply:
It didn't cause issues because it was getting fixed in the kustomize generation but I just fixed it for good measure, please take another look @nojnhuh